yfinance/base.py

```diff
@@ -282,14 +282,15 @@ def cleanup(data):
         # holders
         url = "{}/{}/holders".format(self._scrape_url, self.ticker)
         holders = _pd.read_html(url)
-        self._major_holders = holders[0]
-        self._institutional_holders = holders[1]
-        if 'Date Reported' in self._institutional_holders:
-            self._institutional_holders['Date Reported'] = _pd.to_datetime(
-                self._institutional_holders['Date Reported'])
-        if '% Out' in self._institutional_holders:
-            self._institutional_holders['% Out'] = self._institutional_holders[
-                '% Out'].str.replace('%', '').astype(float)/100
+        if holders and len(holders) >= 2:
```

Comment:
_holders_ usually contains the Major Holders data. I suggest to to check len(holders) > 0 first and then len(holders) > 1

Reply:
https://finance.yahoo.com/quote/WTR/holders actually has no holders.
_holders = _pd.read_html(url)_ will return a _ValueError_ which should be caught too.

Reply:
@tino-247 Thanks. Added this